<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSelectTransactionsDlg</class>
 <widget class="QDialog" name="KSelectTransactionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xxx</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="m_description">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>xxx</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KMyMoneyRegister::Register" name="m_register" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="rowCount" stdset="0">
      <number>0</number>
     </property>
     <property name="columnCount" stdset="0">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyRegister::Register</class>
   <extends>QWidget</extends>
   <header>register.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KSelectTransactionsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KSelectTransactionsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
